package.json

```diff
@@ -56,6 +57,7 @@
     "https-browserify": "^1.0.0",
     "isomorphic-dompurify": "^0.19.0",
     "jsonwebtoken": "^8.5.1",
+    "jszip": "^3.10.1",
```

Comment:
Is this actually still needed? (/Can you tell I scrolled past some of the bigger files for now!)

Reply:
Yep, still used in `github-utils`